docs/introduction/connect.md

```diff
@@ -7,6 +7,11 @@ In Cerebral you connect state to components where you need it. This give some be
 3. Increased readability as every component explicitly tells you what state and signals it needs and where it gets it from
 4. You can safely move the component wherever you want without breaking chain of props passing
 
+Cerebral supports numerous view layers. They conceptually work the same way, but has different implementation details. Choose the view layer that makes sense to you and your team. We will move on using **React**, but have a look at the API section to find more out about **Inferno**, **AngularJS**, **Preact** and **Vue**.
+
+**Cerebrals documentation uses React!**
```

Comment:
Why to mention it if Cerebral documentation is static generated website without controller?